fast_dp/image_readers.py

```diff
@@ -5,6 +5,8 @@
 
 from fast_dp.image_names import image2template_directory
 
+from dxtbx.model.experiment_list import ExperimentList
```

Comment:
Should this be `from dxtbx.model.experiment_list import ExperimentListFactory`?

Reply:
Although it does eventually call through to `ExperimentListFactory`, [from_file](https://github.com/cctbx/dxtbx/commit/973a1515846364487fcfba68e963ff3df1a1d2b8) and [from_templates](https://github.com/cctbx/dxtbx/commit/06aecbf70ea01095f97ba9c0101cc651447082ab) have been on plain `ExperimentList` for a while.

If this needs to support pre-python-2.7 deployments then this probably needs to be changed.